advanced_flask/routes/supermarkets/utils.py

```diff
@@ -0,0 +1,21 @@
+import json
+from json import JSONDecodeError
+
+
+def get_supermarkets_data():
+    with open("routes/supermarkets/supermarkets.json") as file:
+        return json.load(file)

```

Comment:
do not use python reserved words like `file`

Reply:
Also, catch possible errros like `File not found` and reruen empty data if you catch